code/game/objects/effects/traps.dm

```diff
@@ -0,0 +1,28 @@
+/obj/effect/trap
```

Comment:
I refactored mines to be extremely similar to this code unknowingly, one of them should probably be brought under the other's typepath.